src/cswinrt/main.cpp

```diff
@@ -321,7 +321,15 @@ Where <spec> is one or more of:
                 group.add([&componentActivatableClasses, &projectionFileWritten]
                 {
                     writer wm;
-                    wm.write("// This file was generated by cswinrt.exe version %", VERSION_STRING);
+                    wm.write(R"(//------------------------------------------------------------------------------
+// <auto-generated>
```

Comment:
Does the auto-generated tag cause the code analysis tooling to not pick it up?  Is that documented somewhere?

Reply:
I can't immediately find the public docs for it, but the Roslyn code is here: https://github.com/dotnet/roslyn/blob/cf55f3a58e47298426fa971d3bd9d8857c746c65/src/Compilers/Core/Portable/InternalUtilities/GeneratedCodeUtilities.cs